overview.md

```diff
@@ -15,22 +13,22 @@ The Endgame Plan remains in active development, led by Rune Christensen. The lat
 
 The Endgame Plan has many goals which come together to create the Endgame State. A list of these goals is below:
 - Providing access to an Unbiased World Currency in the form of the decentralized stablecoin Dai.
-- Ensure that Maker has a meaningful charitable impact.
 - Ensure that Maker Core is more resilient to external risks such as regulatory action.
 - Ensure that the Maker Ecosystem can scale in a decentralized manner.
-- Alignment engineering for all participants in the Maker Ecosystem resulting in positive community culture and engagement, known as good meta.
+- Alignment engineering for all participants in the Maker Ecosystem resulting in positive community culture and engagement.
 - Reduce governance overhead for MKR holders and Maker delegates.
 - Cut scope creep in Maker Core.
+- Ensure that Maker has a meaningful charitable impact.
 - Reduce MKR concentration through revised tokenomics and MKR emissions.
 
```

Comment:
Lists look better when consistent language is chosen. 
Most of them are on infinitive. 
Change lines...
15 - "Provide"
18 - "Provide Alignment Engineering ..." (Alignment Engineering is capitalized throughout the Atlas except for two exceptions - We should also make this consistent on the Atlas)